roles/ceph-validate/tasks/check_devices.yml

```diff
@@ -24,19 +24,20 @@
   register: dedicated_device_parted
   with_items: "{{ dedicated_devices }}"
   when:
-    - osd_scenario == 'non-collocated'
+    - dedicated_devices|default([]) | length > 0
```

Comment:
not sure to understand this change, because it was only for non-collocated scenario.